{"uuid":"c0ee5599-5974-4fd7-9a32-33f280b45b33","historyId":"596837d0bb8ed91cee2f0d7d7727e6c9","fullName":"APi_Negative.Accounting.SG_NG_Admin.particular_user","labels":[{"name":"package","value":"APi_Negative.Accounting.SG_NG_Admin"},{"name":"testClass","value":"APi_Negative.Accounting.SG_NG_Admin"},{"name":"testMethod","value":"particular_user"},{"name":"parentSuite","value":"All Test Suite"},{"name":"suite","value":"APi_Negative"},{"name":"subSuite","value":"APi_Negative.Accounting.SG_NG_Admin"},{"name":"host","value":"DESKTOP-L0TTVHB"},{"name":"thread","value":"18000@DESKTOP-L0TTVHB.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"This story belongs to the particular_User"}],"links":[],"name":"05:Negative value","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [400] but found [500]","trace":"java.lang.AssertionError: expected [400] but found [500]\r\n\tat org.testng.Assert.fail(Assert.java:110)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1577)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:149)\r\n\tat org.testng.Assert.assertEquals(Assert.java:131)\r\n\tat org.testng.Assert.assertEquals(Assert.java:655)\r\n\tat org.testng.Assert.assertEquals(Assert.java:665)\r\n\tat APi_Negative.Accounting.SG_NG_Admin.particular_user(SG_NG_Admin.java:133)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:580)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1597)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:829)\r\n\tat org.testng.TestRunner.run(TestRunner.java:602)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:437)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Negative value","steps":[],"attachments":[],"parameters":[{"name":"arg3","value":"05"},{"name":"arg2","value":"Negative value"},{"name":"arg1","value":"400"},{"name":"arg0","value":"-21293"}],"start":1739534935817,"stop":1739534937485}